BigBaseV2/src/script_mgr.hpp

```diff
@@ -4,6 +4,19 @@
 
 namespace big
 {
+
+	inline bool g_script_self = true;
+	inline bool g_script_weapons = true;
+	inline bool g_script_vehicles = true;
+	inline bool g_script_misc = true;
+	inline bool g_script_remote = true;
+	inline bool g_script_noclip = true;
+	inline bool g_script_lscustoms = true;
+	inline bool g_script_vehiclefly = true;
+	inline bool g_script_rgbrandomizer = true;
+	inline bool g_script_contextmenu = true;
+	inline bool g_script_turnsignal = true;
```

Comment:
In general I dislike having global variables like these but for now I guess it is fine.

No need to change it as of now.

Reply:
This can be greatly optimized, i admit.